tools/ci-cdk/canary-runner/src/run.rs

```diff
@@ -42,9 +44,12 @@ lazy_static::lazy_static! {
     // This is a map of SDK release tag to smithy-rs commit hash
     static ref PINNED_SMITHY_RS_VERSIONS: Vec<(ReleaseTag, &'static str)> = {
         let mut pinned = vec![
-            // Versions <= 0.6.0 no longer compile against the canary after this commit in smithy-rs
+            // Versions <= 0.6.0 no longer compiles against the canary after this commit in smithy-rs
```

Comment:
I think `compile` was fine here ("versions no longer compile").

Reply:
oops, you are correct, fixed